modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk/query_processor_test.clj

```diff
@@ -23,16 +23,32 @@
    [metabase.util.honeysql-extensions :as hx]
    [toucan2.tools.with-temp :as t2.with-temp]))
 
+(def ^:private test-db-name (bigquery.tx/normalize-name :db "test_data"))
+
+(def ^:private sample-dataset-name (bigquery.tx/normalize-name :db "sample_dataset"))
+
+(defn- with-test-db-name
+  "Replaces instances of v3_test_data with the full per-test-run DB name"
+  [x]
+  (cond
+    (string? x) (str/replace x "v3_test_data" test-db-name)
+    (map? x)    (update-vals x with-test-db-name)
+    (vector? x) (mapv with-test-db-name x)
+    (list?   x) (-> x vec with-test-db-name list*)
```

Comment:
If we want to keep the original types, then we need
```suggestion
    (list?   x) (apply list (map with-test-db-name x))
```
but this wouldn't handle sequences. Maybe we should do
```suggestion
    (seq?    x) (map with-test-db-name x)
```
instead. (We can put it into `doall` if fully realizing is important.)

Reply:
This is only to accommodate test expressions in this file, and the tests pass, so I don't think we need to cover any other cases.

Reply:
And this is just for test equality, so the types don't matter.

```
(= (list 1 2 3) (list* (list 1 2 3))) ;; => true
```